test/functional/test-amp-img.js

```diff
@@ -400,57 +388,63 @@ describe('amp-img', () => {
      *     class that allows it to be a blurred placeholder.
      */
     function getImgWithBlur(addPlaceholder, addBlurClass) {
-      el = document.createElement('amp-img');
-      img = document.createElement('img');
+      const el = document.createElement('amp-img');
+      const img = document.createElement('img');
       if (addPlaceholder) {
         img.setAttribute('placeholder', '');
+        el.getPlaceholder = () => {return img;};
+      } else {
+        el.getPlaceholder = sandbox.stub();
       }
       if (addBlurClass) {
         img.classList.add('i-amphtml-blur');
       }
       el.appendChild(img);
       el.getResources = () => Services.resourcesForDoc(document);
-      el.getPlaceholder = sandbox.stub();
-      el.togglePlaceholder = sandbox.stub();
-      impl = new AmpImg(el);
+      const impl = new AmpImg(el);
       impl.layoutWidth_ = 200;
+
+      impl.togglePlaceholder = sandbox.stub();
+      toggleExperiment(impl.win, 'blurry-placeholder', true, true);
```

Comment:
You only need to call this once in `beforeEach`. `impl.win == window`, so just do:

```js
toggleExperiment(window, 'blurry-placeholder', true, true);
```